app/controllers/users_controller.rb

```diff
@@ -15,4 +15,8 @@ def update
       render :edit
     end
   end
+
+  def bids_placed
```

Comment:
So, you think I should move this back to the `bids_controller` then and map `/account/bids` to the `bids#index` method?